ansible/files/yum.repos.d/centos7.8.2003/cortx.repo

```diff
@@ -0,0 +1,6 @@
+[releases_cortx]
+# Shailesh will give separate URL for centos 7.8.2003 cortx build, for now using same as 7.7.1908
```

Comment:
pls update the correct url for centos 7.8 ..
pls check with Shailesh

Reply:
If its more efforts then we can skip this part for now .. 
it will be safe to use CentOS 7.7 'last_successful' rpms

Reply:
Shailesh has given us new URL ,and I have updated it.